types/node/v4/index.d.ts

```diff
@@ -1,6 +1,8 @@
 // Type definitions for Node.js 4.2
 // Project: http://nodejs.org/
-// Definitions by: Microsoft TypeScript <http://typescriptlang.org>, DefinitelyTyped <https://github.com/DefinitelyTyped/DefinitelyTyped>
+// Definitions by: Microsoft TypeScript <http://typescriptlang.org>,
+//                 DefinitelyTyped <https://github.com/DefinitelyTyped/DefinitelyTyped>
+//                 Marc Ghorayeb <https://github.com/marcghorayeb>
```

Comment:
I vote for `none`, because readers are first interested in the definitions, not in the list of contributors. 
`git` already keeps this information.